lib/Group_LDAP.php

```diff
@@ -573,9 +573,10 @@ private function getGroupsByMember($dn, &$seen = null) {
 			return [];
 		}
 		$seen[$dn] = true;
+		$escapedDn = $this->access->getConnection()->getLDAP()->escape($dn, null, LDAP_ESCAPE_FILTER);
```

Comment:
any additional place where this "escape" function should be used?

Reply:
As I understand, the dn only needs to be escaped when used as a attribute value, in a filter. Which is only the case when searching for members of a group in this case.